package.json

```diff
@@ -26,15 +26,9 @@
   ],
   "scripts": {
     "clean": "rm -rf lib && rm -rf dist && rm -f tsconfig.tsbuildinfo",
-    "build": "yarn build-lib && yarn build-types",
-    "build:release": "yarn clean && yarn build && yarn build-web",
-    "build-lib": "babel src -x .ts -d lib",
-    "build-types": "tsc --declaration --incremental --outDir lib --project tsconfig.build.json --emitDeclarationOnly",
-    "build-web": "webpack --mode production --entry ./lib/web.js --output ./dist/bls.min.js",
-    "check-types": "tsc --noEmit",
```

Comment:
you still need this because build doesn't check types in tests

Reply:
In CI we run `yarn build` which checks types. Also when running mocha test ts-node if src has bad type tests won't run

Reply:
`yarn build` doesn't check types in test/ directory since it uses tsconfig.build.json which has excluded test/ directory.
Are you sure node-ts runs same typescking? Afaik, it runs minimal typechecking for possible runtime errors, might be wrong though.

Reply:
- `yarn build` will typecheck all types in src
- `yarn test` will typecheck only the tests to be run and all the code that they import. If you have a test utility that's not used by anything then it won't be typecheck. In my opinion that's okay since that utility should be removed in such a case